src/consts/models.tsx

```diff
@@ -24,6 +24,9 @@ export const CONST = {
   pin: 'pin',
   defaultLanguage: 'en',
   maxAddressLength: 48,
+  tcVersionRequired: 1,
+  tcValue: 'tcValue',
+  tcVersion: 'tcVersion',
```

Comment:
I think I don't like the shortcut `tc`. I had to think twice to figure out what it means.